pilot/pkg/xds/bench_test.go

```diff
@@ -638,6 +639,24 @@ func BenchmarkCache(b *testing.B) {
 			c.Get(key)
 		}
 	})
+	b.Run("concurrent", func(b *testing.B) {
+		c := model.NewXdsCache()
+
+		key := makeCacheKey(1)
+		req := &model.PushRequest{Start: zeroTime.Add(time.Duration(1))}
+		c.Add(key, req, res)
+		wg := sync.WaitGroup{}
+		wg.Add(10)
+		for worker := 0; worker < 10; worker++ {
+			go func() {
```

Comment:
is the test result stable?  The scheduling of go routines maybe uncontrollable. 

Since you have only 10 routines, the latency compared to the inner loop maybe minor